chimedb/core/alpenhorn.py

```diff
@@ -0,0 +1,32 @@
+"""Alpenhorn-2 integration.
+
+Provides an extension module for alpenhorn-2.
+"""
+
+import peewee as pw
+
+from . import connectdb
+from .exceptions import ConnectionError, NoRouteToDatabase
+
+
+def connect(config):
+    """connectdb.connect() boilperplate"""
```

Comment:
You could probably be a bit more helpful here. It's probably also worth saying that config doesn't do anything.